doc/user_guide/recordflux_ug/04-specification_language.rst

```diff
@@ -0,0 +1,1312 @@
+.. _Specification_Language:
+
+Specification Language
+~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
This looks like a copy of an outdated version of the language reference. I think it would be better to keep the language reference as a separate document and just reference it from the user's guide.